### ISSUE TITLE
Is this repo still being maintained? 

### ISSUE THREAD

**Issue by lukeedward12** [nasa/CFS_CI#4]
It has been several years since any updates were pushed to this repository and I am wondering whether that will change. I have tried building this against a newer version of a cFS Bundle (https://github.com/nasa/cFS) and it doesn't seem to be compatible (cFE APIs and conventions have changed). I have noticed in that in the CTF releases (https://github.com/nasa/CTF/releases/tag/v1.8.2), they include a sample cFS workspace with an updated (or forked?) CI app that is compatible with newer version of the cFS Bundle. 

Will this repository be updated in the future? Is this still being maintained?

**Comment by lukeedward12** [comment]
@jtovias

**Comment by MichaelBonnet** [comment]
this doesn't appear to be maintained, for the reasons you mentioned. The API is fully outdated.

**Comment by lukeedward12** [comment]
Yeah, this is one of the core issues with cFS today. There is not enough funding at NASA directed towards the cFS-verse, so these bespoke applications in the "App Store" often fall behind the cFE and are not maintained. 

We can discuss how to rectify **that** issue, but for your use now I would recommend just manually updating API calls or using the CTF release. I've had to do this with several cFS "Apps".

**Comment by MichaelBonnet** [comment]
> Yeah, this is one of the core issues with cFS today. There is not enough funding at NASA directed towards the cFS-verse, so these bespoke applications in the "App Store" often fall behind the cFE and are not maintained.
> 
> We can discuss how to rectify **that** issue, but for your use now I would recommend just manually updating API calls or using the CTF release. I've had to do this with several cFS "Apps".

Are you saying you've gotten the CTF release's CI and TO to work in a modern (draco-rc4/draco-rc5) cFE release?
